weblate/trans/views/settings.py

```diff
@@ -339,18 +339,28 @@ def announcement_delete(request: AuthenticatedHttpRequest, pk):
 
 @login_required
 def component_progress(request: AuthenticatedHttpRequest, path):
-    obj = parse_path(request, path, (Component,))
+    """
+    Show component update progress.
```

Comment:
shouldn’t it be `Shows component....`? Not sure if this is a name of a feature or a description.

Reply:
It's 'imperative mood' docstring. Doing otherwise will trigger [error D401](https://docs.astral.sh/ruff/rules/non-imperative-mood/) from ruff checks.